lib/TH/THHalf.h

```diff
@@ -24,17 +24,26 @@ typedef struct __thalign__(4) {
 typedef __THHalf THHalf;
 typedef __THHalf2 THHalf2;
 
-/* numeric limits */
+typedef unsigned short THHalfBits;
```

Comment:
tbh I'd probably just get rid of the typedef; we don't typedef the actual definition, so it doesn't buy us much.  And I don't think we'd want to typedef the definition given ffi/general complexity.